plugins/modules/problem_task.py

```diff
@@ -0,0 +1,435 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2021, XLAB Steampunk <steampunk@xlab.si>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+DOCUMENTATION = r"""
+module: problem_task
+
+author:
+  - Manca Bizjak (@mancabizjak)
+  - Miha Dolinar (@mdolin)
+  - Tadej Borovsak (@tadeboro)
+
+short_description: Manage ServiceNow problem tasks
+
+description:
+  - Create, delete or update a ServiceNow problem tasks
+  - For more information, refer to the ServiceNow problem management documentation at
+    U(https://docs.servicenow.com/bundle/paris-it-service-management/page/product/problem-management/concept/c_ProblemManagement.html).
+
+version_added: 1.0.0
+
+extends_documentation_fragment:
+  - servicenow.itsm.instance
+  - servicenow.itsm.sys_id.info
+  - servicenow.itsm.number.info
```

Comment:
Are these two OK? I would assume we need regular ones here since we are not in the info module.

```suggestion
  - servicenow.itsm.sys_id
  - servicenow.itsm.number
```